zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -342,20 +362,15 @@ public boolean runParagraph(String noteId,
       callback.onFailure(new IOException("paragraph is disabled."), context);
       return false;
     }
-    p.setText(text);
-    p.setTitle(title);
-    p.setAuthenticationInfo(context.getAutheInfo());
-    p.settings.setParams(params);
-    p.setConfig(config);
-
-    if (note.isPersonalizedMode()) {
```

Comment:
I think the second paragraph in the following two paragraphs of code is repeated.
```
    // First paragraph
    p.setText(text);
    p.setTitle(title);
    p.setAuthenticationInfo(context.getAutheInfo());
    p.settings.setParams(params);
    p.setConfig(config);

    // Second paragraph
    if (note.isPersonalizedMode()) {
      p = note.getParagraph(paragraphId);
      p.setText(text);
      p.setTitle(title);
      p.setAuthenticationInfo(context.getAutheInfo());
      p.settings.setParams(params);
      p.setConfig(config);
    }
```
Because whether note.isPersonalizedMode() is equal to true or false,
```
    p.setText(text);
    p.setTitle(title);
    p.setAuthenticationInfo(context.getAutheInfo());
    p.settings.setParams(params);
    p.setConfig(config);
```
All executed.  So I deleted the second paragraph of duplicate code.
Please help me review here. Thank!